### ISSUE TITLE
Is it appropriate to test results with part of target trained with labels ?

### ISSUE THREAD

**Issue by lyxok1** [thuml/Transferable-Query-Selection#4]
Hi,
     Thanks for sharing the code. Here I have a concern after reading the code. In the Line271-275 in `main.py`, we see the selected labeled samples are appended to source data set to be trained with labels.

```
source_train_ds1.add_item(active_samples)
source_train_ds2.add_item(active_samples)
source_train_ds3.add_item(active_samples)
source_train_ds4.add_item(active_samples)
source_train_ds5.add_item(active_samples)
```

However, I did not see the operation to remove these samples from test set like `target_test.remove_item(active_samples)`, that is, in the test set, there will be some samples (5% in this code) where labels of them are already known by the training process, and it is easy for models trained with labels to categorize these test samples with near 100% accuracy, thus increase the overall performance in unlabeled data of target domain. I did not delve too deep into the field of Active Learning, so I wonder is such setting (labeled datasets overlap with unlabeled one) appropriate for  researches in this field ? 

Appreciation if you can reply my concerns.

**Comment by microhhh** [comment]
When the labeled data is relatively small, the difference in accuracy is very small, and it mainly depends on the accuracy of the remaining 95% data. In actual use, these two evaluation methods are reasonable. 
In addition, in the statistical active learning method, the accuracy of the training set is generally difficult to reach 100%.

**Comment by lyxok1** [comment]
Thanks for your reply~
